src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -103,8 +103,9 @@ public CCDCaseApi(
     }
 
     public List<Claim> getByDefendantId(String id, String authorisation) {
+        LOGGER.debug("ccd search is by authorisation instead of defendant id {}", id);
         User user = userService.getUser(authorisation);
-        return getAllCasesBy(user, ImmutableMap.of("case.defendantId", id));
+        return getAllCasesBy(user, ImmutableMap.of());
```

Comment:
Is this intentional?

Reply:
ccd list cases by authorisation, means cases authorised to user and authorisation has id as well. it is not required because defendantId is beried in case.defendants[].defendantId,  filter on it not possible straight  away

Reply:
Thanks